src/stan/model/indexing/lvalue.hpp

```diff
@@ -141,15 +140,15 @@ inline void assign(Vec1&& x,
   if (idxs.head_.is_ascending()) {
     const auto slice_start = idxs.head_.min_ - 1;
     const auto slice_size = idxs.head_.max_ - slice_start;
-    stan::math::check_size_match("vector[min_max] assign sizes", "lhs and rhs",
+    stan::math::check_size_match("vector[min_max] assign", "left hand side",
                                  slice_size, name, y.size());
     x.segment(slice_start, slice_size) = y;
     return;
   } else {
     const auto slice_start = idxs.head_.max_ - 1;
     const auto slice_size = idxs.head_.min_ - slice_start;
-    stan::math::check_size_match("vector[reverse_min_max] assign sizes",
-                                 "lhs and rhs", slice_size, name, y.size());
+    stan::math::check_size_match("vector[reverse_min_max] assign",
+                                 "left hand side", slice_size, name, y.size());
```

Comment:
I changed this to left hand side so that the error would read something like 

```
"vector[reverse_min_max] assign: left hand side (500) and ANON (4) must match in size" 
```